cibyl/sources/elasticsearch/client.py

```diff
@@ -0,0 +1,51 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+
+import logging
+
+from elasticsearch import Elasticsearch
+
+from cibyl.exceptions.elasticsearch import ElasticSearchError
+
+LOG = logging.getLogger(__name__)
+
+
+class ElasticSearchClient:
+
+    def __init__(self: object,
+                 host: str = "http://localhost",
+                 port: int = 9200) -> None:
+        """Initialization of ElasticSearchClient
```

Comment:
I think you need to move the docstring from __init__ to the class and change little bit the description. At least this is what I see here: https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

Reply:
I was curious about this  and I found that according to [pep 257](https://www.python.org/dev/peps/pep-0257/) this is correct. See:

> The docstring for a class should summarize its behavior and list the public methods and instance variables. If the class is intended to be subclassed, and has an additional interface for subclasses, this interface should be listed separately (in the docstring). The class constructor should be documented in the docstring for its __init__ method. Individual methods should be documented by their own docstring.

In summary, if we decide to follow this pep, host and port in the init docstring, address and a general description of the class in the class one

Reply:
We can decide in the next meeting how to follow this. Thx.